static/src/javascripts/projects/common/modules/experiments/tests/play-video-on-fronts.js

```diff
@@ -20,17 +27,58 @@ define([
 
         this.canRun = function () {
             // Only videos that are links to video pages have data-embed-paths
-            return config.page.isFront && $('.fc-item--has-video-main-media .js-video-play-button').length > 0;
+            // Or article pages that have been linked to from test-enhanced cards
+            return (config.page.isFront && $('.fc-item--has-video-main-media .js-video-play-button').length > 0) ||
+                (config.page.contentType === 'Article' && hasAutoplayHash);
```

Comment:
This would never have been able to have been true as it's from the added functionality i.e. Adding the link to the play button, so it won't really affect the numbers.